cognite/client/_api/data_modeling/instances.py

```diff
@@ -68,8 +68,8 @@ def as_ids(self) -> list[NodeId | EdgeId]:
 
 
 class _NodeOrEdgeResourceAdapter:
-    @classmethod
-    def _load(cls, data: str | dict, cognite_client: Optional[CogniteClient] = None) -> Node | Edge:
+    @staticmethod
+    def _load(data: str | dict, cognite_client: Optional[CogniteClient] = None) -> Node | Edge:
```

Comment:
Not a fan of `classmethods`?

Reply:
Definitely a fan 😉 But these methods doesn't use `cls` and thus removing the unused argument simplifies reasoning about them